src/main/host/process.rs

```diff
@@ -191,7 +191,8 @@ pub struct RunnableProcess {
     // Shared memory allocation for shared state with shim.
     shim_shared_mem_block: ShMemBlock<'static, ProcessShmem>,
 
-    strace_logging: Option<StraceLogging>,
+    // strace logging file, if any. Shared with forked Processes.
+    strace_logging: Option<RootedRc<RootedRefCell<StraceLogging>>>,
```

Comment:
Why is the `RootedRefCell` needed? I don't think we need to mutate the format options at runtime, and the file is already inside a `RefCell`. If it's needed for `Send`/`Sync` or something, maybe replace the `RefCell` in `StraceLogging` with `RootedRefCell` instead?

Reply:
Right, otherwise the inner `RefCell` can't go inside the `RootedRc`, since it's not `Sync`. I'll take a look at replacing the inner `RefCell` instead if `File::try_clone` doesn't work out.